browser/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: ['mixmax', 'plugin:react/recommended'],
+  parser: 'babel-eslint',
+  settings: {
+    react: {
+      version: '15.5.4'
```

Comment:
it will customize the errors/warnings based on the react version we specify [eg](https://github.com/yannickcr/eslint-plugin-react/blob/ecc327a63a779deb625414c61e617798ad912dbf/docs/rules/no-deprecated.md).

hm, yes, it could be annoying to update in lockstep. if we omit the version, the plugin will default to the latest-stable version (which we're using now and would want to continue using, probably), maybe we should do that?

i'm having a bit of trouble testing that since i've `link`ed my local `eslint-config-mixmax`, but i can dig deeper if you think it's worth it